Dockerfile

```
@@ -19,5 +21,8 @@ VOLUME ["/var/lib/ldap", "/etc/cesldap"]
 # LDAP PORT
 EXPOSE 389
 
+# healtcheck
+HEALTHCHECK CMD [ $(doguctl healthy ldap; echo $?) == 0 ]
```

Good you thought of the health check! :heart:

You provided an older healthcheck-style which works fine. Anyhow, this health check can be simplified more (some IDE don't like the superfluous shell syntax)

```suggestion
HEALTHCHECK CMD doguctl healthy ldap || exit 1
```

---

I guess we have to change some/all other dogus dockerfiles, too.

---

We already do. It is mentioned in our DoD